app/main/menu.js

```diff
@@ -423,7 +423,7 @@ class AppMenu {
 		const resetAppSettingsMessage = 'By proceeding you will be removing all connected organizations and preferences from Zulip.';
 
 		// We save App's settings/configurations in following files
-		const settingFiles = ['window-state.json', 'domain.json', 'settings.json', 'certificates.json'];
+		const settingFiles = ['window-state.json', 'config/domain.json', 'config/settings.json', 'config/certificates.json'];
```

Comment:
@abhigyank I think we should move this `window-state.json` file to `/config`.

Reply:
`window-state.json` is created by this module. I don't think we can change where this module creates the file. If we move it to `/config`, it will recreate the file in the appData folder.